benchmarks/lockhammer/src/lockhammer.c

```diff
@@ -55,6 +55,7 @@ void print_usage (char *invoc) {
     fprintf(stderr,
             "Usage: %s\n\t[-t threads]\n\t[-a acquires per thread]\n\t"
             "[-c critical iterations]\n\t[-p parallelizable iterations]\n\t"
+            "[-s]\n\t"
```

Comment:
Should we have more verbose help message, instead of simple [-s]?

```
Usage: [-option VALUE] [-option] ...
[-t threads]                set active thread number, default max
[-a acquires]               set acquires per thread, default 50000
[-c critical]               set critical iterations, default 0
[-p parallel]               set parallelizable iterations, default 0
[-s]                        enable safemode, default no
```